src/neuroglancer/perspective_view/panel.ts

```diff
@@ -225,6 +226,39 @@ export class PerspectivePanel extends RenderedDataPanel {
       });
     });
 
+    registerActionListener(element, 'translate-annoation-via-mouse-drag', (e: ActionEvent<MouseEvent>) => {
+      const {mouseState} = this.viewer;
+      const selectedAnnotationId = mouseState.pickedAnnotationId;
+      const annotationLayer = mouseState.pickedAnnotationLayer;
+
+      let voxelSize = this.viewer.navigationState.voxelSize
+      if (typeof(annotationLayer) != 'undefined'){
+        if (typeof(selectedAnnotationId) != 'undefined'){
+          
+          let annotationRef = annotationLayer.source.getReference(selectedAnnotationId)!;
+          let ann = <Annotation>annotationRef.value
+          
+          const handler = getAnnotationTypeHandler(ann.type)
+          let point = handler.getClosestPoint(ann, voxelSize, mouseState.position)
```

Comment:
For bounding boxes, it would be great if it were possible to drag any of the 8 corners.  However, admittedly that would add some complexity.